packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -92,14 +93,38 @@ export class MetricExporter implements IMetricExporter {
     metrics: MetricRecord[],
     cb: (result: ExportResult) => void
   ): Promise<void> {
+    return this._exportAsync(metrics)
```

Comment:
This function doesn't need to be async anymore

Reply:
Which function?

Reply:
Sorry, the `export()` function, you can just remove the `async` since it doesn't await anymore